assignment/Final_Proposal.md

```diff
@@ -0,0 +1,95 @@
+# Final Project Proposal
+
+Look into the Ridership Data of the Divvy Bike-sharing System in Chicago
+
+## Problem / Question
+
+Being a perfect activity to personal health and natural environment, biking has
+always been an ideal activity for people to commute and exercise. In large cities,
+bike sharing becomes more and more popular among citizens and tourists because of
+the convenience, affordability, and sustainability of the bike-sharing system.
+The City of Chicago has its city-wide bike-sharing system called Divvy, started
+in 2007 by the Chicago Department of Transportation. Now Divvy operates 6,000 bikes
+and more than 570 stations across Chicago, providing Chicagoans with a convenient,
+healthy, and affordable transportation alternative. Although the stations of Divvy
+are located across Chicago, the city center has a much higher density of location
+than other parts of the city. However, is every Divvy stop in Chicago really working
+as we expect? How can we evaluate the performance of each Divvy station?
+
+To evaluate the overall performance of the Divvy bike-sharing stations, I decide to
+develop a website to map the historic trip counts of each Divvy station within the
+city boundary of Chicago. Then pick out the top 10 stations with the greatest daily
+trip counts, as well as the bottom 10 stations with the lowest daily trip counts.
+
+Although trip counts cannot comprehensively represent the performance level of each Divvy station,
+it can reflect the station's daily supply of bike-sharing service from the perspective
+of trip count.
+
+## The data
+
+API: https://data.cityofchicago.org/resource/fg6s-gzvg.geojson
+Historic Divvy Trips (2013 to present)
+This lists individual Divvy bike sharing trips, including the origin, destination,
+and timestamps for each trip. Trips are included when there is an starting and end date.
+Trips using a subscriber pass will include some basic demographic data (gender and age)
+that is associated with the account.
+
+## Technologies used
+
+API: map trip data stored in API and filter the dataset to only show trips occurring in 2018
+https://data.cityofchicago.org/resource/6zsd-86xi.json?$where=date between '2018-01-01T00:00:00' and '2019-01-01T00:00:00'
+
+jQuery: when the user clicks any Divvy station on the map, the website will return the
+average daily trip counts through jQuery functions, such as  $().click and $().change.
+
+underscore: use underscore functions to calculate the average daily trip counts of
+each Divvy station, such as map/each.
+
+mapbox/turf/leaflet: I'm not sure what I'll do with these technologies, but I'm sure
```

Comment:
Off the top of my head, it might be useful to know how many other bike stations are within a given radius (or even better, you could use a mapbox isochrone)